src/explore-education-statistics-frontend/src/components/Feedback.tsx

```diff
@@ -0,0 +1,191 @@
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import styles from '@frontend/components/Feedback.module.scss';
+import Button from '@common/components/Button';
+import classNames from 'classnames';
+import { useMemo, useState } from 'react';
+import FormProvider from '@common/components/form/FormProvider';
+import { Form, FormFieldTextArea } from '@common/components/form';
+import ButtonGroup from '@common/components/ButtonGroup';
+import feedbackService from '@frontend/services/feedbackService';
+import {
+  FeedbackRequest,
+  FeedbackResponse,
+} from '@common/services/types/feedback';
+import Yup from '@common/validation/yup';
+import { ObjectSchema } from 'yup';
+
+type BannerState = 'initial' | 'notUseful' | 'problemEncountered' | 'thanks';
+
+interface FormValues {
+  context?: string;
+  issue?: string;
+  intent?: string;
+}
+
+export default function Feedback() {
+  const [bannerState, setBannerState] = useState<BannerState>('initial');
+  const [response, setResponse] = useState<FeedbackResponse | undefined>();
+
+  const handleUsefulFeedback = async () => {
+    await feedbackService.sendFeedback({
+      url: window.location.pathname,
+      userAgent: navigator.userAgent,
+      response: 'Useful',
+    });
+
+    setBannerState('thanks');
+  };
+
+  const cancel = () => {
+    setResponse(undefined);
+    setBannerState('initial');
+  };
+
+  const handleSubmit = async (values: FeedbackRequest) => {
+    await feedbackService.sendFeedback({
+      url: window.location.pathname,
+      userAgent: navigator.userAgent,
+      response: response as FeedbackResponse,
+      context: values.context,
+      issue: values.issue,
+      intent: values.intent,
+    });
+
+    setBannerState('thanks');
+  };
+
+  const feedbackLimit = 2000;
+
+  const validationSchema = useMemo<ObjectSchema<FormValues>>(() => {
+    return Yup.object({
+      context: Yup.string().max(
+        feedbackLimit,
+        `Description must be ${feedbackLimit} characters or less`,
```

Comment:
Error messages should be uniquely identifiable in the error summary. Currently, if we have max length errors in both fields we end up with the following: 

![image](https://github.com/user-attachments/assets/2738984c-3d98-407d-9174-2d459e2cf4b0)

It's unclear which error corresponds to which field, which is confusing if you try and click on one of the error links.

Would suggest we change this to 'What were you doing must be 2000 characters or less'.

A similar thing can be done for the other messages below:

- What were you hoping to achieve must be 2000 characters or less
- What went wrong must be 2000 characters or less